docs/source/packages.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
This rst file is needed for Javadocs to appear in the toc. I think there's a formatting issue in the index.rst of master that's preventing packages.rst to be picked up. Can we remove this orphan directive?